app/src/Doctrine/Website/RST/Kernel.php

```diff
@@ -32,7 +34,17 @@ public function getName()
 
     public function getDirectives()
     {
-        return array_merge($this->baseKernel->getDirectives(), $this->directives);
+        $directives = array_merge($this->baseKernel->getDirectives(), $this->directives);
+
+        foreach ($directives as $key => $directive) {
+            if (! ($directive instanceof CodeBlock)) {
+                continue;
+            }
+
+            $directives[$key] = new CodeBlockDirective();
```

Comment:
What is replacing these values achieving?

Reply:
This was a temporary hack while I was working on getting everything working. It won't be necessary once I am done.